mac/macdeploy.sh

```diff
@@ -7,6 +7,7 @@
 
 MACDEPLOYQT="macdeployqt"
 LIBRARY_NAME="libQupZilla.2.dylib"
+PLUGINS="QupZilla.app/Contents/Resources/plugins"
```

Comment:
Is this really the correct path? I think this is "our" Resources path - eg. for application. Libraries (Frameworks) should be in different path.